packages/jest-each/src/index.js

```diff
@@ -1,16 +1,25 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ */
+
 import bind from './bind';
 import arrayEach from './array';
 import templateEach from './template';
 
-const each = (...args) => {
+const each = (...args: any) => {
```

Comment:
Can we use `mixed` type here?

Reply:
I’m AFK at the moment, I’ll update this later tonight when I get home or in another PR I’ve got a few refactors/improvements I want to make with the each code :smile:

Reply:
Oh, I think `Array<mixed>` should do the trick here, cause `args` is an array. Or we can be fancy and use `$ReadOnlyArray<*>`

Reply:
Nice just updated :)